choose/options.go

```diff
@@ -13,6 +13,7 @@ type Options struct {
 	CursorPrefix      string       `help:"Prefix to show on the cursor item (hidden if limit is 1)" default:"[•] " env:"GUM_CHOOSE_CURSOR_PREFIX"`
 	SelectedPrefix    string       `help:"Prefix to show on selected items (hidden if limit is 1)" default:"[✕] " env:"GUM_CHOOSE_SELECTED_PREFIX"`
 	UnselectedPrefix  string       `help:"Prefix to show on unselected items (hidden if limit is 1)" default:"[ ] " env:"GUM_CHOOSE_UNSELECTED_PREFIX"`
+	Selected          string       `help:"Options that should start as selected" default:"" env:"GUM_CHOOSE_SELECTED"`
```

Comment:
I'm fairly sure if you make this a `[]string` `kong` will parse it correctly, allowing you to skip the `parseSelectedItems` step, making the code a little cleaner.

Reply:
you are correct, I just tested it now and it worked! I am sending the changes right now